docs/Create-Packages/Creating-a-Package.md

```diff
@@ -42,6 +42,7 @@ In this topic:
 - [Setting a package type](#setting-a-package-type) (NuGet 3.5 and later)
 - [Adding a readme and other files](#adding-a-readme-and-other-files)
 - [Including MSBuild props and targets in a package](#including-msbuild-props-and-targets-in-a-package)
+- [Authoring COM Interop packages compatible with Package Reference](#authoring-com-interop-packages-compatible-with-package-reference)
```

Comment:
How about "Including COM interop assemblies in a package"? why specify Package Reference right there anyways authors aren't going to differentiate anything for packages.config or PackageReference. And also, everything in our docs, should be default to PackageReference.